amqphandler/amqphandler_test.go

```diff
@@ -169,8 +166,13 @@ func cleanup() {
 	}
 }
 
-func sendMessage(message interface{}) error {
```

Comment:
What the reason to change this? You have to duplicate send message and expected message in test data.

Reply:
resolves